src/installer/test/Assets/TestProjects/AppWithWait/Program.cs

```diff
@@ -0,0 +1,16 @@
+using System;
```

Comment:
License header?

Reply:
There's no License header for any of the test assets in https://github.com/dotnet/runtime/tree/master/src/installer/test/Assets/TestProjects 

So, I kept the same pattern.

Reply:
Can you please fix that in a followup PR? Or at least create an issue for it.

Reply:
OK I'll fix it for all the projects.